pkg/providers/openstack/get_config.go

```diff
@@ -1,15 +1,11 @@
 package openstack
 
 import (
-	"github.com/emc-advanced-dev/unik/pkg/compilers"
 	"github.com/emc-advanced-dev/unik/pkg/providers"
 )
 
 func (p *OpenstackProvider) GetConfig() providers.ProviderConfig {
 	return providers.ProviderConfig{
 		UsePartitionTables: true,
-		SupportedCompilers: []string{
```

Comment:
Ok, in the previous commit, I added a comment about this one, but now I see it doesn't make sense.